setup.py

```diff
@@ -101,9 +101,9 @@ def get_git_head_sha(base_dir):
 
 def get_build_version(xla_git_sha):
   version = os.getenv('TORCH_XLA_VERSION', '2.2.0')
-  if _check_env_flag('VERSIONED_XLA_BUILD', default='0'):
+  if _check_env_flag('VERSIONED_XLA_BUILD', default='1'):
     try:
-      version += '+' + xla_git_sha[:7]
+      version += '+git' + xla_git_sha[:7]
```

Comment:
hmm, I guess for the release build we need to turn this `VERSIONED_XLA_BUILD ` to 0?

Reply:
Yea, that's correct.

Reply:
we might want to add a new env FLAG `RELEASED_XLA_BUILD` here, such as:
```
if _check_env_flag('VERSIONED_XLA_BUILD', default='1') and _check_env_flag(' RELEASED_XLA_BUILD=1', default='0')::
   try:
...
```

then add `RELEASED_XLA_BUILD=1` in https://github.com/pytorch/xla/blob/master/infra/tpu-pytorch-releases/artifacts.auto.tfvars for public release version

Reply:
Why adding another env? Can't we just use the VERSIONED_XLA_BUILD directly?